sim/hunter/hunter.go

```diff
@@ -89,6 +89,9 @@ type Hunter struct {
 	WingClip       *core.Spell
 
 	Shots []*core.Spell
+	Strikes []*core.Spell
+	MeleeSpells []*core.Spell
+	LastShot *core.Spell
```

Comment:
Since this is set dynamically, you would want to make sure to set it to `nil` in the `Reset` method so that it doesn't leak across iterations